src/modules/dynamic-Todo.js

```diff
@@ -0,0 +1,188 @@
+        dotBtn.setAttribute('data-mdb-ripple', 'true');
+        dotBtn.setAttribute('data-mdb-ripple-color', 'light');
+        dotBtn.setAttribute('data-bs-toggle', 'dropdown');
+        dotBtn.innerHTML = `<svg
+      xmlns="http://www.w3.org/2000/svg"
+      class="h-6 w-6"
+      fill="none"
+      viewBox="0 0 24 24"
+      stroke="currentColor"
+      stroke-width="2"
+      >
+      <path
+      stroke-linecap="round"
+      stroke-linejoin="round"
+      d="M12 5v.01M12 12v.01M12 19v.01M12 6a1 1 0 110-2 1 1 0 010 2zm0 7a1 1 0 110-2 1 1 0 010 2zm0 7a1 1 0 110-2 1 1 0 010 2z"
+      />
+      </svg>`;
+        dotBtn.classList = `dropdown-toggle
+      px-1
+      py-1
+      bg-slate-100
+      text-gray-400
+      font-medium
+      text-xs
+      leading-tight
+      rounded
+      shadow-md
+      hover:bg-slate-300 hover:shadow-lg
+      focus:bg-slate-300 focus:shadow-lg focus:outline-none focus:ring-0
+      active:bg-slate-300 active:shadow-lg active:text-white
+      transition
+      duration-150
+      ease-in-out
+      flex
+      items-center
+      whitespace-nowrap`;
+
+        const dropItemDel = document.createElement('li');
+        const delBtn = document.createElement('button');
+        delBtn.type = 'button';
+        delBtn.innerHTML = `<svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5" fill="none" viewBox="0 0 24 24" stroke="currentColor" stroke-width="2">
+      <path stroke-linecap="round" stroke-linejoin="round" d="M15 12H9m12 0a9 9 0 11-18 0 9 9 0 0118 0z" />
+    </svg> 
+      <span>DEL</span>`;
+        delBtn.classList = `del-btn dropdown-item flex items-center space-x-2
+      text-sm
+      py-2
+      px-4
+      font-normal
+      w-full
+      whitespace-nowrap
+      bg-transparent
+      text-gray-700
+      transition
+      duration-150
+      hover:bg-red-500 hover:text-white`;
+        delBtn.onclick = () => todoInstance.onDelete(id);
+        dropItemDel.appendChild(delBtn);
+
+        const dropItemEdit = document.createElement('li');
+        const editBtn = document.createElement('button');
+        editBtn.type = 'button';
+        editBtn.innerHTML = `<svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5" fill="none" viewBox="0 0 24 24" stroke="currentColor" stroke-width="2">
+      <path stroke-linecap="round" stroke-linejoin="round" d="M11 5H6a2 2 0 00-2 2v11a2 2 0 002 2h11a2 2 0 002-2v-5m-1.414-9.414a2 2 0 112.828 2.828L11.828 15H9v-2.828l8.586-8.586z" />
+    </svg>
+    <span>EDIT</span>`;
+        editBtn.classList = `edit-btn dropdown-item flex items-center space-x-2
+      text-sm
+      py-2
+      px-4
+      font-normal
+      w-full
+      whitespace-nowrap
+      bg-transparent
+      text-gray-700
+      transition
+      duration-150
+      hover:bg-amber-500 hover:text-white`;
+        editBtn.onclick = () => todoInstance.onEdit(id);
+        dropItemEdit.appendChild(editBtn);
+
+        const dropmenu = document.createElement('ul');
```

Comment:
Consider using CamelCase for the variable "dropmenu". This will improve the readability of your code.

Reply:
Good point, on that thank you